spec/javascript/e2e/SupplementalApplicationPage/confirmation_modal.e2e.js

```diff
@@ -36,9 +36,10 @@ describe('SupplementalApplicationPage confirm modal', () => {
     await sharedSteps.goto(page, `/applications/${LEASE_UP_LISTING_APPLICATION_ID}/supplementals`)
 
     page.waitForSelector('.tabs').then(() => page.click('.tabs li:nth-child(1)'))
+    await page.waitForNavigation()
```

Comment:
I think this was the issue, that the page was loading and being closed at the same time. I updated the test so we check that it navigates to the application snapshot page instead of checking that the confirm modal wasn't there

Reply:
:+1: